### ISSUE TITLE
Tautological example in Set-SFRemoteLoggingHost

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#66]
From get-help:

```
    -------------------------- EXAMPLE 1 --------------------------
    
    PS C:\>$rh = Get-SFRemoteLoggingHost
    Set-SFRemoteLoggingHost -RemoteHosts $rh -Confirm:$false
    
    Sets the RemoteLoggingHost with a LoggingServer with no confirmation prompt.

```

If one starts with nothing (in $rh), he sets the value to nothing. It's not clear how the array looks like and how it should be composed. 

With no idea how the input looks like, it took me a long while to figure this out... Maybe Example 1 could be changed to use this or a new example could be added

```
PS C:\>$syslogServer = @{"Host" = "192.168.1.100"; "Port" = 514;}
PS C:\>Set-SFRemoteLoggingHost -RemoteHosts $syslogServer

Host          Port
----          ----
192.168.1.100  514

```

**Comment by Salmanius** [comment]
Thank you Sean. I've added a task for us to update/rewrite this example in our 1.6 release.

**Comment by arjun960** [comment]
@scaleoutsean Help of Set-SFRemoteLoggingHost is updated and the change is available in the Powershell SDK 12.3 release. So closing the issue.
